Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:6.0
+// swift-tools-version:6.1
```

Comment:
Not required in this pull request, but some cleanup is needed:

- `git grep -i 'version.*\b6\.0'`
- `git grep 'compiler('`
- `README.md`
- `GRDB.swift.podspec`